src/components/sidebar/tabs/AssetsSidebarTab.vue

```diff
@@ -43,14 +43,15 @@
     <template #body>
       <div v-if="displayAssets.length" class="relative size-full">
         <VirtualGrid
-          v-if="displayAssets.length"
+          v-if="!loading && displayAssets.length"
           :items="mediaAssetsWithKey"
```

Comment:
nit: The different conditionals in here are a little hard to track.
Normally I'd expect a top level loader (or error state), then a fallthrough to the contentful item.

Also, I think the second `displayAssets.length` is redundant since if we get to the VirtualGrid, the outer div already guarantees that we have some items.

Reply:
Beautiful, thanks!